src/api-client.ts

```diff
@@ -0,0 +1,114 @@
+import * as httpm from "@actions/http-client";
+
+const userAgent = "GitHub multi-repository variant analysis";
```

Comment:
Should this include information about the fact that we're calling from the Action workflow? E.g. `GitHub multi-repository variant analysis action`